ci-operator/jobs/openshift/builder/openshift-builder-presubmits.yaml

```diff
@@ -181,6 +177,9 @@ presubmits:
               name: ci-operator-openshift-builder
         - name: RPM_REPO
```

Comment:
@smarterclayton the ci-operator should provide this since we have `tag_specification` ...

Reply:
RPM_REPO today means "rpms for this repository".  I'm hesitant to default RPM_REPO to the release artifact version of an unrelated system.  

We could maybe argue this is OPENSHIFT_RPM_REPO which the template accepts, and if the template doesn't want to do that, it just serves the artifacts from a pod that it itself launches.  That becomes easier when the ci-operator is creating the templates (someone just reuses "host RPMS for this image").

Reply:
huh? in what case should the RPM repo be different from that which is served from the release we are tagging in?

Reply:
The RPMs used by the install always come from origin, not the team repos